hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/TopicEndpoint.java

```diff
@@ -63,6 +52,16 @@
     TopicMetrics getMetrics(@PathParam("topicName") String qualifiedTopicName);
 
     @POST
+    @Produces(APPLICATION_JSON)
+    @Path("/{topicName}/blacklist")
```

Comment:
I thought about it, but didn't want to multiply endpoint classes for that sake. It's still a topic related action. Do you prefer `TopicBlacklistEndpoint` ?